key-vault.tf

```diff
@@ -1,18 +1,29 @@
-module "shared-vault" {
+module "old_shared_vault" {
   source = "git@github.com:hmcts/cnp-module-key-vault.git?ref=master"
   name = "ccd-shared-${var.env}"
   product = "${var.product}"
   env = "${var.env}"
   tenant_id = "${var.tenant_id}"
   object_id = "${var.jenkins_AAD_objectId}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
+  resource_group_name = "${azurerm_resource_group.shared_rg.name}"
+  product_group_object_id = "be8b3850-998a-4a66-8578-da268b8abd6b"
+}
+
+module "new_shared_vault" {
```

Comment:
I would advise against using this name for the vault resource. Because when we want to rename it later to something that doesn't include `new` it's going to cause the resource to be destroyed and recreated at which point it will be empty once more.

Instead, I would advise picking a long-lasting name and sticking to it.

My name suggestion for the resource would simply be `vault`. Given we are in `ccd-shared-infrastructure`, the shared part is already part of the context.

So the old vault can be kept with it's existing name `shared-vault` and the new one will be `vault`.

Reply:
Nice, will do